### ISSUE TITLE
Enemies disappear near lab elevator 

### ISSUE THREAD

**Issue by viewerusa** [CleverRaven/Cataclysm-DDA#38684]
# Describe the bug
Enemies, including experimental mutants, zombie scientists, and possibly all enemies vanish from the game when they're on or near a lab elevator being used by the player. 

# Steps To Reproduce
1. Use a lab elevator to descend from the surface.
2. Draw enemies close or on to the elevator. Take note of the enemies you see in the room.
3. Take the elevator back to the surface.
4. Descend on the elevator again.
5. Take note again of the enemies you now see and compare to before.

# Expected behavior
I expected the enemies to be in more or less the same position after taking a quick trip to the surface and back, since a round trip on it only takes seconds ingame time. 

# Screenshots
![lab1](https://user-images.githubusercontent.com/62019373/76316032-3c150c80-62b0-11ea-966f-1f7a6f53c4cb.PNG)
![lab2](https://user-images.githubusercontent.com/62019373/76316046-420aed80-62b0-11ea-820e-0cb009fc9992.PNG)
![lab3](https://user-images.githubusercontent.com/62019373/76316052-43d4b100-62b0-11ea-8de7-1a17173a954d.PNG)
![lab4](https://user-images.githubusercontent.com/62019373/76316059-48996500-62b0-11ea-8d0f-26ecf6227258.PNG)

# Versions and configuration

- OS: Windows
    - OS Version: 10.0 1903
- Game Version: 0.D-12846-gd972c1c [64-bit]
- Graphics Version: Tiles
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food]
 - Ingame language: English

**Comment by Night-Pryanik** [comment]
Z-levels turned on?

**Comment by Amoebka** [comment]
The next time you walk close to stairs anywhere in the world, the "disappeared" monsters will come out of them.

**Comment by viewerusa** [comment]
You're right, the lab entrance is right next to a town and now i see the mutants on the edge of the screen fighting some zombies.
